### ISSUE TITLE
Check out dextr

### ISSUE THREAD

**Issue by adirbru** [adirbru/USPrediction#6]
1. Deep Learning-based Video Propagation Tools
🔹 Deep Extreme Cut (DEXTR) + MaskTrack or MaskProp
Start with one labeled frame

Use optical flow and attention-based mask propagation

Often used in medical imaging tasks

Pretrained versions may require fine-tuning on US data

Implementation: [MaskTrack](https://github.com/wasidennis/MaskTrack) or [MaskPropagation](https://github.com/ndrplz/maskprop)

**Comment by eyalamdur** [comment]
Not usefull enough for our project
